manim/mobject/svg/svg_mobject.py

```diff
@@ -76,6 +77,10 @@ def __init__(
         should_subdivide_sharp_curves=False,
         should_remove_null_curves=False,
         color=None,
+        *,
```

Comment:
Is this a typo or intentional?

Reply:
This is intentional, so as not to change the constructor signature. `fill_color`, `stroke_color` and `stroke_opacity` were keyword only before and they are keyword only now.